waspc/src/Wasp/AI/GenerateNewProject/Plan.hs

```diff
@@ -56,6 +56,8 @@ generatePlan newProjectDetails planRules = do
     appDescriptionBlockText = appDescriptionBlock newProjectDetails
     basicWaspLangInfoPrompt = Prompts.basicWaspLangInfo
     waspFileExamplePrompt = Prompts.waspFileExample
+    waspPlanExample = Prompts.waspPlanExample
+    waspPlanSchema = Prompts.waspPlanSchema
```

Comment:
While I don't terribly mind you extracting these, I am not sure what is the purpose of it? In Common/Prompts.hs we keep common prompts, that are used in multiple places -> these prompts here are not used in multiple places. I get that it might feel attractive to have all prompts in one place, but then we lose the structure we had with all the Haskell files, and all of them are in one file and are harder in my opinion to work with. We could have alternative hierary inside Prompts/, but I wouldn't do that.

Reply:
Extracting these seemed convenient because then the structure of `planPrompt` was easier for me to understand. However, I missed the part where fields in `Prompts` are used in multiple places. 

As you suggested, we could probably talk about extracting `waspPlanExample` and `waspPlanSchema` into other files, but I think we agree that that's not a priority right now.

I'll go ahead and un-extract these.

Reply:
Makes sense!

Tip again regarding commenting in PRs: what would make this the easiest for me here is if you put the :rocket: on intiail comment and didn't resolve this.

Then I see the :rocket: and I know change was done (right now I am not sure, I will have to go check it in the code, because you said you will do it but I am not sure if that happened yet).
Also, I read you comment and then I resolve, while if you resolved it, I would normally miss it completely.